src/lib/y2storage/btrfs_subvolume.rb

```diff
@@ -241,29 +240,23 @@ def former_referenced_limit
     # @return [Boolean]
     def require_default_mount_point?
       return false unless filesystem.root?
-      return false if top_level? || default_btrfs_subvolume? || snapshot?
+      return false if top_level? || default_btrfs_subvolume? || for_snapshots?
 
       parent_subvolume.top_level? || parent_subvolume.prefix?
     end
 
-    # Whether the subvolume is for snapshots
+    # Whether the subvolume is used for snapshots
     #
     # @return [Boolean]
-    def snapshot?
-      path.match?(/^.snapshots/)
+    def for_snapshots?
+      path.match?(/.snapshots/)
```

Comment:
Should this rely on `Filesystems::Btrfs#snapshots_root` instead of a hadcoded ".snapshots"?

Reply:
Yes, in fact, there was a corner case with this hypothetical scenario:

~~~
# top-level                       
# |-- @                           
#    |-- @/.snapshots               
#        |-- @/.snapshots/1/snapshot 
# |-- .snapshots

subvol1.path            #=> @/.snapshots/1/snapshot
subvol1.for_snapshots?  #=> true

subvol2.path            # => .snapshots
subvol2.for_snapshots?  # => true (but it should be false)
~~~